packages/tools/pie-css/scss/settings/_breakpoints.scss

```diff
@@ -0,0 +1,19 @@
+// Interval settings to be used by @include-media Sass library
+// https://eduardoboucas.github.io/include-media/documentation/#variable-unit-intervals
+$unit-intervals: (
+    'px': 1,
+    'em': 0.001, // We need more sensitivity, some browsers are very specific with breakpoints
+    'rem': 0.1,
+    '': 0
+);
+
+// Breakpoints to be used by @include-media Sass library
+// https://eduardoboucas.github.io/include-media/documentation/#variable-breakpoints
+$breakpoints: (
+    xs: 320px,
+    sm: 600px,
+    md: 768px,
+    lg: 1024px,
```

Comment:
I would prefer `s`, `m`, `l`, but not sure if there's a reason why not to

Reply:
After our conversation last week with @thejfreitas, I'll keep it as it is.

[Bootstrap](https://getbootstrap.com/docs/5.0/layout/breakpoints/#available-breakpoints) and [Material](https://mui.com/material-ui/customization/breakpoints/) both use similar breakpoint keys, so may feel more familiar to consumers.